app/controllers/submissions_controller.rb

```diff
@@ -1,6 +1,14 @@
 class SubmissionsController < ApplicationController
   authorize_resource
 
+  def show #show form to move submission to different code review
+    @submission = Submission.find(params[:id])
+    @similar_code_reviews = Course.internship_courses.current_and_future_courses.map {|course| course.code_reviews}.flatten.select {|cr| cr.title.strip == @submission.code_review.title.strip}
```

Comment:
This is a lot of logic for a controller!

Reply:
What about `@submission.similar_code_reviews`?

Reply:
Agreed about a lot of logic. I thought about adding it to the model; it's just that this is the only time I'll ever be using this method, and it seems oddly specific to have in the model? If you think it would be better in the model, I'll gladly move it there. Thanks!

Reply:
It's still a good idea to have it in the model, even if it's only used here. Any time you can move logic from the controller to the model it's usually a good idea. Then you can unit test it more easily and if the logic changes it's encapsulated in the model.